Modules/IO/NIFTI/include/itkNiftiImageIO.h

```diff
@@ -133,8 +158,8 @@ class ITKIONIFTI_EXPORT NiftiImageIO : public ImageIOBase
    * the nifti library maintainers.  This format does not properly respect the file orientation fields.
    * By default this is set to true.
```

Comment:
Need to update the method documentation: default is not "true", also add some more details (summary of the discussions, reason for default value choice)

Reply:
oops , looks like i forgot to set the default to AnalyzeITK4Warning